### PR TITLE
Print transcripts so unicode is printed properly

### PR DESCRIPTION
**Pull request by jerjou**

### CODE REVIEW

File: speech/api-client/transcribe.py [hunk 1]
```diff
@@ -75,7 +75,15 @@ def main(speech_file):
     # [END construct_request]
     # [START send_request]
     response = service_request.execute()
+
+    # First print the raw json response
     print(json.dumps(response))
```
Comment (jonparrott) [inline speech/api-client/transcribe.py]: might as well go ahead and add `indent=2` to this?

Comment (jerjou) [inline speech/api-client/transcribe.py]: Ah yes. I like.

**Comment by puneith** [comment]
Sorry I am late. But why not print(json.dumps(item, encoding='utf8', ensure_ascii=False))?

**Comment by jerjou** [comment]
Oh. Yup. That woulda worked too. I totally didn't think of the available parameters to json.dumps, clearly -_-;

**Comment by jonparrott** [comment]
Send another PR?

**Pull request merged**